src/view/configuration/configuration.jsx

```diff
@@ -39,10 +39,11 @@ const contextGranularityEnum = {
 };
 const contextOptions = ["web", "device", "environment", "placeContext"];
 
-const instanceDefaults = {
+const getInstanceDefaults = initInfo => ({
```

Comment:
We now need `initInfo` when getting the default values for an instance because that's where the user's active org ID comes from.